modules/exploits/linux/http/dlink_850l_unauth_exec.rb

```diff
@@ -0,0 +1,229 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'openssl'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'DIR-850L (Un)authenticated OS Command Exec',
+      'Description' => %q{
+        This module leverages an unauthenticated credential disclosure
+        vulneralbility to then execute arbitrary commands on DIR-850L routers
+        as an authenticated user. Unable to use Meterpreter payloads.
+      },
+      'Author' => [
+        'Mumbai <https://github.com/realoriginal>' # module
```

Comment:
Can you move the URL to the comment? That field is normally for an e-mail address, so it can mess up the parsing. :/

Reply:
Also, I think you should credit Zdenda for the bugs unless you are that person. :)